docs/source/DeveloperGuide/Internals/Core/componentsAuthNandAuthZ.rst

```diff
@@ -40,3 +40,60 @@ Two decorators are available for safely doing all that:
 
   * :py:func:`~DIRAC.Core.Utilities.Proxy.executeWithoutServerCertificate`
   * :py:func:`~DIRAC.Core.Utilities.Proxy.executeWithUserProxy`
+
+
+================================
+Authentication and authorization
+================================
+When a client calls a service, he needs to be identified. If a client opens a connection a :py:class:`~DIRAC.Core.DISET.private.Transports.BaseTransport` object is created then the service use the handshake to read certificates, extract informations and store them in a dictionnary so you can use these informations easily. Here an example of possible dictionnary::
+
+	{
+		'DN': '/C=ch/O=DIRAC/[...]',
+		'group': 'devGroup',
+		'CN': u'ciuser', 
+		'x509Chain': <X509Chain 2 certs [...][...]>, 
+		'isLimitedProxy': False, 
+		'isProxy': True
+	}
+
+
+When connection is opened and handshake is done, the service calls the :py:class:`~DIRAC.Core.DISET.AuthManager` and gave him this dictionnary in argument to check the authorizations. More generally you can get this dictionnary with :py:meth:`BaseTransport.getConnectingCredentials <DIRAC.Core.DISET.private.Transports.BaseTransport.BaseTransport.getConnectingCredentials>`.
+
+
```

Comment:
The following paragraphs are very useful and interesting, but they probably should not be in the file, and not in this section, which is the Developer guide. Rather they should be in the administrator guide. 
I think that it could be a new page, linked from http://dirac.readthedocs.io/en/integration/AdministratorGuide/index.html just after "DIRAC configuration" and before "DIRAC systems in details".

You can also take something to add from https://docs.google.com/presentation/d/1eTo8E5fYjqZ0i54sgYlnna7JPxhTZgZaGSmq-Ts1afs/edit?usp=sharing (slides 8, 9, 10).

Reply:
Ok, I'm will take a look at your presentation to add informations and i will move these paragraphs.